Header header
geometry_msgs/Pose2D lower_left_point
geometry_msgs/Pose2D upper_right_point
geometry_msgs/Pose2D center


